chain-impl-mockchain/src/accounting/account/spending.rs

```diff
@@ -0,0 +1,155 @@
+//! Spending strategies
+use super::LedgerError;
+
+/// Simple strategy to spend from multiple increasing counters
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct SpendingCounterIncreasing {
+    nexts: Vec<SpendingCounter>,
+}
+
+// number of bits reserved for lanes
+const LANES_BITS: usize = 3;
+// number of bits reserved for counter (unrespective of the lane)
+const UNLANES_BITS: usize = 32 - LANES_BITS;
+
+impl SpendingCounterIncreasing {
+    /// number of parallel lanes of increasing counters
+    pub const LANES: usize = 1 << LANES_BITS;
```

Comment:
Not sure if useful in this case, but now that we have `const generics` it could be implemented with them?
```rust
impl<const LANES_BITS: usize> SpendingCounterIncreasing  {
    pub const LANES: usize = 1 << LANES_BITS;
    ...
}
```

Reply:
it's not terribly useful for anything though, if it's not hardcoded here, it would be very likely in the user, otherwise it will be snowball'ed to every type in the ledger

Reply:
We could split it in a generic implementation and a type alias for the specific lenght.

```rust
type SpendingCounterIncreasing = GenericSpendingCounterIncreasing<3>;
```
But it is ok anyway.